### PR TITLE
feat: initial configuration for grafana dashboard

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Preview:

image

image

image

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

## Refer to a related PR or issue link (optional)

https://github.com/GreptimeTeam/greptimedb/issues/3262

### CODE REVIEW

File: config/grafana/greptimedb.json [hunk 1]
```diff
@@ -0,0 +1,2541 @@
...
+              "id": "byNames",
+              "options": {
+                "mode": "exclude",
+                "names": [
+                  "{instance=\"localhost:4000\", job=\"greptime\"}"
+                ],
+                "prefix": "All except:",
+                "readOnly": true
+              }
+            },
```
Comment (evenyag) [inline config/grafana/greptimedb.json:1933]:
```suggestion
```

File: config/grafana/greptimedb.json [hunk 2]
```diff
@@ -0,0 +1,2541 @@
...
+    "from": "now-30m",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "GreptimeDB",
+  "uid": "e7097237-669b-4f8d-b751-13067afbfb68",
+  "version": 6,
+  "weekStart": ""
+}
```
Comment (evenyag) [inline config/grafana/greptimedb.json:2541]:
```suggestion
}

```

Comment (waynexia) [reply config/grafana/greptimedb.json]: it doesn't have tailing empty line

**Review by evenyag — APPROVED** [review]

File: grafana/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Grafana dashboard for GreptimeDB
+--------------------------------
+
+GreptimeDB's official Grafana dashboard.
```
Comment (killme2008) [inline grafana/README.md:4]: Adds `WIP`? We are still working in progress.

**Review by killme2008 — APPROVED** [review]
LGTM

File: grafana/README.md (hunk 3 above)
Comment (waynexia) [inline grafana/README.md:4]:
```suggestion
GreptimeDB's official Grafana dashboard.

Status notify: we are still working on this config. It's expected to change frequently in the recent days. Please feel free to submit your feedback and/or contribution to this dashboard 🤗
```

**Pull request merged**